lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -697,7 +710,7 @@ void HdVP2BasisCurves::_UpdateDrawItem(
 
     HdDirtyBits itemDirtyBits = drawItem->GetDirtyBits();
 
-    CommitState                    stateToCommit(*drawItem);
```

Comment:
Minor signature change. We're commit data for a render item, so the CommitState doesn't get access to the drawItem anymore.